fluent-react/package.json

```diff
@@ -50,14 +50,15 @@
     "prop-types": "^15.6.0"
   },
   "peerDependencies": {
+    "jsdom": "^10.0.0 || ^11.0.0",
```

Comment:
Why support both v10 and v11?

Reply:
Actually, should we add this here at all? For the SSR use-case, `jsdom` is more of an optional peer dependency and I'm worried this will result in warnings printed by `npm install fluent-react`.